plotter/views.py

```diff
@@ -818,14 +846,192 @@ def export(request):
+            if tosplitbed == '':
+                continue
+            tosplitbedlist = tosplitbed.split('\n')
+            tosortbedlist = []
+            for line in tosplitbedlist:
+                if line != '':
+                    tosortbedlist.append(line.split(' '))
+            tounsplitbedlist = sorted(tosortbedlist, key=lambda lambdaline: (unicode(lambdaline[0]), int(lambdaline[1])))
+            unsplitbedlist = []
+            for line in tounsplitbedlist:
+                unsplitbedlist.append('\t'.join(line))
+            tosavebed = '\n'.join(unsplitbedlist)
+            if datatype != "segments":
+                bedfile = open(x+"_"+datatype+'.bed', 'w')
+            else:
+                bedfile = open(x+"_"+datatype+'.bedGraph', 'w')
+            bedfile.write(tosavebed)
+            bedfile.close()
+            usedlist.append(datatype)
+            if datatype != "segments":
+                subprocess.call(['bedToBigBed', x+"_"+datatype+'.bed', 'chrom.sizes', x+"_"+datatype+'.bigbed'])
+            else:
+                subprocess.call(['bedGraphToBigWig', x + "_segments.bedGraph", 'chrom.sizes', x + "_segments.bigWig"])
+            if x != "ES0004" and datatype != "segments":
+                os.remove(x+"_"+datatype+'.bed')
+            elif x != "ES0004":
+                os.remove(x+"_segments.bedGraph")
+        bedgraphsecretfilelocation = db.secret_file(x+'.bedGraph.gz')
+        copy2(bedgraphsecretfilelocation, os.getcwd()+'/'+x+'.bedGraph.gz')
+        gzipfile = gzip.open(os.getcwd()+'/'+x+'.bedGraph.gz')
+        gzipfile.next()
+        bedgraphdata = gzipfile.read()
+        gzipfile.close()
+        os.remove(os.getcwd()+'/'+x+'.bedGraph.gz')
+        bedgraph = open(x+'.bedGraph', 'w')
+        bedgraph.write(bedgraphdata)
+        bedgraph.close()
+        subprocess.call(['bedGraphToBigWig', x+'.bedGraph', 'chrom.sizes', x+'.bigWig'])
+        os.remove(x+'.bedGraph')
+    trackdbtxt = open('trackDb.txt', 'w')
+    for x in request.POST['profile']:
+        trackdbtxt.write('track ' + x + 'multiWig\ntype bigWig\ncontainer multiWig\naggregate transparentOverlay\n' +
+                         'shortLabel ' + request.POST['short_label'] + 'multiWig\nlongLabel ' +
+                         request.POST['long_label'] + 'multiWig\nautoScale on\nnegativeValues on\nalwaysZero on\n\n')
+        for datatype in usedlist:
+            if datatype != "segments":
+                trackdbtxt.write('track ' + x + "_" + datatype + '\nbigDataUrl ' + x+'_'+datatype+'.bigbed' +
+                                 '\nshortLabel ' + request.POST['short_label'] + datatype + '\nlongLabel ' +
+                                 request.POST['long_label'] + datatype + '\ntype bigBed\ncolor 0,253,0' +
+                                 '\nalwaysZero on\n\n')
+        trackdbtxt.write('track ' + x + '\nbigDataUrl ' + x + '.bigWig\nshortLabel ' + request.POST['short_label'] +
```

Comment:
is there some way to remove repetition in these trackdbtxt.write lines?

also instead of using `'track ' + x + "_"` you may consider using `'track %s_'.format(x)` etc